Client/Frontend/Contextual Hint/ContextualHintViewController.swift

```diff
@@ -77,7 +77,15 @@ class ContextualHintViewController: UIViewController, OnViewDismissable {
 
     // MARK: - Properties
     private var viewModel: ContextualHintViewModel
-    private var onViewSummoned: (() -> Void)?
+
+    private lazy var onViewSummoned: (() -> Void)? = { [self] in
+        switch viewModel.hintType {
+        case .jumpBackIn, .jumpBackInSyncedTab, .toolbarLocation:
+            BrowserViewController.foregroundBVC().homePanelDidPresentContextualHintOf(type: viewModel.hintType)
```

Comment:
There is probably a better way to handle this. The hint view controller shouldn't be aware of the browser view controller since it is the object that both created it and owns the reference to it.
One option would be to use delegation. Since BVC directly owns the hint view it should be easy to wire up.